### ISSUE TITLE
Click handlers

### ISSUE THREAD

**Issue by flaviusdemian** [wxxsw/GSMessages#9]
Hello,

Can you put  on the right side a 2 buttons? So that we have a message with actions - "Tap to see", "YES" OR "NO"

**Comment by wxxsw** [comment]
This is an example:

``` swift
let message = GSMessage(text: "Hello World!", type: .Success, options: [.TextAlignment(.Left)], inView: view, inViewController: self)

let yes = UIButton(type: .Custom)
    yes.backgroundColor = UIColor.blueColor()
    yes.setTitle("YES", forState: .Normal)
    yes.frame = CGRect(x: 200, y: 7, width: 50, height: 30)

let no = UIButton(type: .Custom)
    no.backgroundColor = UIColor.redColor()
    no.setTitle("NO", forState: .Normal)
    no.frame = CGRect(x: 260, y: 7, width: 50, height: 30)

message.messageView.addSubview(yes)
message.messageView.addSubview(no)
message.show()
```

*Note: Update to the latest version. *

By the way, you should use AutoLayout, not the frame.

Good Luck!

**Comment by flaviusdemian** [comment]
Thank you @wxxsw

**Comment by aranasaurus** [comment]
Something must have changed since this was posted or I'm missing something. `GSMessage` has no `public` initializers, so I'm unclear on how to get at the view to inject subviews into.

**Comment by wxxsw** [comment]
@aranasaurus I'm sorry, this is my fault. I submitted the code and released the new version [1.3.4](https://github.com/wxxsw/GSMessages/tree/1.3.4).

The following demo is the same as above, but uses `Swift3` and `SnapKit`:

```swift
let message = GSMessage(text: "Hello World!",
                        type: .success,
                        options: [.textAlignment(.left)],
                        inView: view,
                        inViewController: self)
        
let yes = UIButton(type: .custom)
    yes.backgroundColor = UIColor.blue
    yes.setTitle("YES", for: .normal)

let no = UIButton(type: .custom)
    no.backgroundColor = UIColor.red
    no.setTitle("NO", for: .normal)

message.messageView.addSubview(yes)
message.messageView.addSubview(no)

yes.snp.makeConstraints { make in
    make.centerY.equalTo(message.messageText)
    make.right.equalTo(no.snp.left).offset(-10)
    make.size.equalTo(CGSize(width: 50, height: 30))
}

no.snp.makeConstraints { make in
    make.centerY.equalTo(message.messageText)
    make.right.equalTo(-10)
    make.size.equalTo(CGSize(width: 50, height: 30))
}

message.show()
```

**Comment by aranasaurus** [comment]
@wxxsw makes sense, thanks for the update! Unfortunately I'm stuck on Swift 2.x for this update, so I'll probably just fork and fix it until I can migrate to Swift 3 in the next update.

**Comment by aranasaurus** [comment]
@wxxsw here's the changeset: https://github.com/aranasaurus/GSMessages/tree/public-api-swift-2 just in case you wanted to do a branch/tag (1.2.4 maybe?) from it.

**Comment by wxxsw** [comment]
@aranasaurus Yes, I have released [1.2.4](https://github.com/wxxsw/GSMessages/tree/1.2.4) and uploaded it to CocoaPods.

**Comment by aranasaurus** [comment]
@wxxsw awesome! thanks!
